tests/installation/disk_boot.pm

```diff
@@ -0,0 +1,35 @@
+# SUSE's openQA tests
```

Comment:
Don't we already have a `disk_boot` module?

Reply:
Yes, I had some errors with the imports as that one is meant to be used with microos distri, so I decided to copy the module, with only the necessary bits

Reply:
Urgh, can we get one that works on both? That microos is so far removed from TW is IMO a problem on its own

Reply:
I guess so, let me see what can I come up with, the issue was with `!main_micro_alp::is_dvd()` that could not be resolved under `opensuse` but can't recall well anymore

https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/f6511f3e08f7d642a249a344aba60e0d9008e58c/tests/microos/disk_boot.pm#L26

Reply:
@Vogtinator looking at the module again, I think both modules are doing different things `microos/boot_disk` expects a system that doesn't have graphical login, adding the code to make it work for both, would have more unintended consequences (same as modifying i.e `installation/first_boot`

Reply:
Can we somehow unify it? IMO having two modules with the same name which do different things makes it even worse

Reply:
I think it would be better to rename it then to e.g `simple_boot` or something since `microos/disk_boot` inherits from consoletest, it expects a console: http://10.149.209.202/tests/140#step/disk_boot/9 which makes it trickier to unify both modules.